### PR TITLE
Common: removed retired dev networks

### PR DESCRIPTION
**Pull request by emersonmacro**
Addresses #1294

### CODE REVIEW

**Review by holgerd77 — CHANGES_REQUESTED** [review]
Hi there,
thanks for the PR! 😄 

For the failing tx tests this is a bit laborious, but an upgrade is needed there. I also once stumbled upon this. This just needs a `Common` instance with the `london` HF activated, so this is not directly related to `aleut`. The thing is: once you change the network there some comparison values in the tests will also change, e.g. the hash in the `hash()` test case, since the hash function also takes the chain ID from the tx into account and the chain ID changes when the network changes.

So on these test cases you need to update - e.g. on the `hash()` test the expected hash to the new hash produced by `signed.hash()` (for now you can assume that this new hash is correct).

Also note: please use a stable/lasting network (like `mainnet` or `rinkeby` for replacement), if you use `calaveras` (the new dev network) e.g., the test will need some adoption again further down the line once *this* network is again removed. 😋

Ok, hope this helps, let me know if you have questions.

File: packages/common/src/genesisStates/index.ts [hunk 1]
```diff
@@ -7,19 +7,13 @@ const genesisStates: genesisStatesType = {
...
     '123': 'calaveras',
   },
   mainnet: require('./mainnet.json'),
   ropsten: require('./ropsten.json'),
   rinkeby: require('./rinkeby.json'),
   kovan: require('./kovan.json'),
   goerli: require('./goerli.json'),
-  yolov3: require('./yolov3.json'),
-  aleut: require('./aleut.json'),
-  baikal: require('./baikal.json'),
```
[the remark this answers is not in the document]
Comment (emersonmacro) [reply packages/common/src/genesisStates/index.ts:22]: Ah, I missed that there were separate json files with the same names there. PR updated, files removed.

**Comment by emersonmacro** [comment]
@holgerd77 Thank you - that info was very helpful. I converted the failing `tx` test to use `rinkeby` and tests are passing now

**Review by holgerd77 — APPROVED** [review]
Cool, thanks for working this in so quickly, looks really good now, will merge.

**Pull request merged**